src/server/log.js

```diff
@@ -14,7 +14,6 @@ function error(error) {
     }
     console.log(chalk.red.bold('SIM ERROR: ' + error));
     config.server && config.server.close();
-    process.exit(1);
```

Comment:
I am removing the `server.close` call as well from `log.js`. I added them to `bin/simulate` for command line consumption scenarios. As @guillaumejenkins said, programmatic consumers should handle errors directly.